policyengine_us/tests/policy/baseline/gov/states/mo/tax/income/deductions/mo_pension_and_ss_or_ssd_deduction/integration_tests/mo_pension_and_ss_or_ssd.yaml

```diff
@@ -371,3 +371,50 @@
         state_code: MO
   output:  # expected results from patched TAXSIM35 2023-02-13 version
     mo_income_tax: 3095.77
+
+- name: Tax unit with taxsimid 94115 in f21.its.csv and f21.ots.csv
+  absolute_error_margin: 0.01
+  period: 2021
+  input:
+    people:
+      person1:
+        is_tax_unit_head: true
+        age: 69
+        employment_income: 15_010
+        taxable_interest_income: 5_505
+        taxable_pension_income: 3_000
+        social_security: 1_500
+        ssi: 0  # not in TAXSIM35
+        state_supplement: 0  # not in TAXSIM35
+        wic: 0  # not in TAXSIM35
+      person2:
+        is_tax_unit_spouse: true
+        age: 69
+        employment_income: 3_010
+        taxable_interest_income: 5_505
+        taxable_pension_income: 3_000
+        social_security: 1_500
+        ssi: 0  # not in TAXSIM35
+        state_supplement: 0  # not in TAXSIM35
+        wic: 0  # not in TAXSIM35
+      person3:
+        age: 11
+        ssi: 0  # not in TAXSIM35
+        state_supplement: 0  # not in TAXSIM35
+        wic: 0  # not in TAXSIM35
+    spm_units:
+      spm_unit:
+        members: [person1, person2, person3]
+        snap: 0  # not in TAXSIM35
+        tanf: 0  # not in TAXSIM35
+    tax_units:
+      tax_unit:
+        members: [person1, person2, person3]
+        premium_tax_credit: 0  # not in TAXSIM35
+    households:
+      household:
+        members: [person1, person2, person3]
+        state_code: MO
+  output:  # expected results from hand calculations using 2021 MO Form 1040
+    mo_pension_and_ss_or_ssd_deduction_section_b: [1_485, 1_485, 0]
```

Comment:
Not necessary for this PR but do you think it makes sense to replace the `ss_or_ssd` in these parameters and variables with `ss` or `oasdi`?

Reply:
Yes.   See code-health issue #1793.